helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -49,6 +49,7 @@ receivers:
   {{- if (eq (include "splunk-otel-collector.metricsEnabled" .) "true") }}
   hostmetrics:
     collection_interval: 10s
+    root_path: {{ .Values.isWindows | ternary "C:\\hostfs" "/hostfs" }}
```

Comment:
Q.: is it safe to assume that is always installed on the `C:` drive? Sorry, I had to ask...

Reply:
it was set like this. I don't know any better.

Reply:
oh, if I am understanding this right you must set it to empty on anything other than Linux, see https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/0f2fea60e36e9951a3caf4884a8264473a8204c5/receiver/hostmetricsreceiver/hostmetrics_others.go#L14

Reply:
good catch, I'll fix.